js/stores/appStore.js

```diff
@@ -238,14 +238,17 @@ const createWindow = (action) => {
     mainWindow.webContents.on('did-finish-load', (e) => {
       lastEmittedState = appState
       mainWindow.webContents.setZoomLevel(zoomLevel[toolbarUserInterfaceScale] || 0.0)
-      e.sender.send(messages.INITIALIZE_WINDOW,
-        {
+
+      const mem = muon.shared_memory.create({
```

Comment:
Not in 0.18.x, this PR is against 0.18.x, so it wouldn't show this diff if there was.